### PR TITLE
[toast-messages] more precise language for org toast messages

### PR DESCRIPTION
**Pull request by dvdherron**
## Cute animal pic
image

_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
https://trello.com/c/sW6isTAD/285-toast-messages-about-scratch-orgs
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x ] Description is in Trello card

## Steps to test/reproduce
Messages about orgs should be specific and say Scratch/Test/Dev depending on the type of org a message is referring to. 
- Create, delete or make changes in an org.
- The toast message should say 'Test', 'Dev' or 'Scratch' org for each respective type.
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me

image

image

image

image

_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by dvdherron** [comment]
[Toast Messages about Scratch Orgs](https://trello.com/c/sW6isTAD/285-toast-messages-about-scratch-orgs)

**Comment by dvdherron** [comment]
@jgerigmeyer Some of the translations still need to be added but I wanted to have you look at what I've done so far in the meantime. 

I think the changes I've made cover all of the Scratch Org toast messages.

**Review by jgerigmeyer — COMMENTED** [review]
@dvdherron This is looking good! I made a few suggestions that I think will simplify the `actions` file a bit, and then I did a quick global find/replace to update a few other places.

File: src/js/store/orgs/actions.ts [hunk 1]
```diff
@@ -107,13 +119,14 @@ export const provisionOrg =
...
-      const { name, parent } = getOrgParent(model, state);
+      const { name, parent, orgType } = getOrgParent(model, state);
+      let msg = i18n.t('Successfully created {{orgType}}.', { orgType });
+
       /* istanbul ignore else */
       if (name && parent) {
         msg = i18n.t(
-          'Successfully created scratch org for {{parent}} “{{name}}”.',
-          { parent, name },
+          'Successfully created {{orgType}} for {{parent}} “{{name}}.”',
```
Comment (jgerigmeyer) [inline src/js/store/orgs/actions.ts:128]: Heh, good catch. We may want to make this change consistent throughout the app? I'll add a note to https://trello.com/c/J04dg5Yi/293-i-want-consistent-capitalization-of-core-objects-terms-eg-project-task-scratch-org

**Review by jgerigmeyer — APPROVED** [review]
🎉

**Comment by jgerigmeyer** [comment]
@dvdherron I think this might be ready for merge -- are there other things missing?

**Comment by dvdherron** [comment]
> 
> 
> @dvdherron I think this might be ready for merge -- are there other things missing?

Fantastic 🎉 

This should cover it. It can be merged @jgerigmeyer

**Pull request merged**